lib/condor.py

```diff
@@ -95,20 +95,24 @@ def load_submit_file(filename):
 
 def submit(f, vargs, schedd_add, cmd_args=[]):
     """Actually submit the job, using condor python bindings"""
-    if vargs["no_submit"]:
+
+    if 'no_submit' in vargs and vargs["no_submit"]:
         print("NOT submitting file:\n%s\n" % f)
         return
     if f:
         print("submitting: %s" % f)
         fl = glob.glob(f)
         if fl:
             f = fl[0]
+
+    print("cmd=args: %s" % cmd_args)
+
     schedd_name = schedd_add.eval("Machine")
     schedd = htcondor.Schedd(schedd_add)
     print("schedd: %s" % schedd_name)
 
     if True:
-        cmd = "/usr/bin/condor_submit -spool -pool %s -remote %s %s %s" % (
+        cmd = "/usr/bin/condor_submit -pool %s -remote %s %s %s" % (
```

Comment:
Why is this change needed?  If I'm looking at this correctly, it'll change current jobsub_submit as well, which we shouldn't do unless we know it's necessary.